lib/concurrent/mutable_struct.rb

```diff
@@ -197,20 +197,20 @@ def []=(member, value)
     end
 
     # @!macro struct_new
```

Comment:
Please update the [`struct_new`](https://github.com/ruby-concurrency/concurrent-ruby/blob/e14ccb3718a256338e55adc6a71970d73c639bfe/lib/concurrent/mutable_struct.rb#L13) definition to include documentation about `keyword_init` option.